### PR TITLE
improve slice validation error messages

### PR DESCRIPTION
**Pull request by danigo99**
## Context

Ticket: https://linear.app/prismic/issue/SMX-137/aauser-i-want-to-see-a-more-descriptive-error-message-when-a-slice

## Impact / Dependencies

The read errors thrown by the next-adapter hook could still be improved:
At the moment reading any slice in a library that contains invalid JSON (even if it's not the invalid slice itself) will log an error. This is due to the hook reading every slice in a library to find the model for the requested one. 
This means that we will still see 1 error for each slice in a library that has invalid JSON.

This PR does not address this issue and will be improved on by @lihbr when updating the next adapter

## Checklist before requesting a review
- [ ] I hereby declare my code ready for review.
- [ ] If it is a critical feature, I have added tests.
- [ ] The CI is successful.
- [ ] If there could backward compatibility issues, it has been discussed and planned.

### CODE REVIEW

File: packages/start-slicemachine/src/StartSliceMachineProcess.ts [hunk 1]
```diff
@@ -219,8 +219,11 @@ export class StartSliceMachineProcess {
 		// Validate Slice models.
 		const allSlices = await this._sliceMachineManager.slices.readAllSlices();
 		if (allSlices.errors.length > 0) {
-			// TODO: Provide better error message.
-			throw new Error(allSlices.errors.join(", "));
+			throw new Error(
+				`Errors ocurred while validating your project's slices.\n\n${allSlices.errors.join(
```
Comment (bapmrl) [inline packages/start-slicemachine/src/StartSliceMachineProcess.ts]: `occurred` 🙂

**Review by bapmrl — COMMENTED** [review]
Nice, but I'm not sure to understand your comment. Could you provide a bit more explanations please? 🙂
>Hook error messages have not been improved here, so we will still see 1 error for each slice in a library that has invalid JSON.

**Review by bapmrl — APPROVED** [review]

File: packages/start-slicemachine/src/StartSliceMachineProcess.ts (hunk 1 above)
Comment (angeloashmore) [reply packages/start-slicemachine/src/StartSliceMachineProcess.ts]: "Slices" should be capitalized as well. 🙂

(I'm just doing a drive-by review. Don't mind me. 😅)

**Review by lihbr — APPROVED** [review]
LGTM

**Pull request merged**